app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java

```diff
@@ -148,7 +148,7 @@ public class ApplicationContext extends Application implements DefaultLifecycleO
     CallMessageProcessor callMessageProcessor;
     MessagingModuleConfiguration messagingModuleConfiguration;
 
-    private volatile boolean isAppVisible;
+    public static volatile boolean isAppVisible;
```

Comment:
We should limit the use of static variable as much as possible. If you change this one into a normal non-static field, you should be able to access it through a normal context, like this:

```kotlin
(context.applicationContext as ApplicationContext).isAppVisible
```

Reply:
Done